x/crosschain/keeper/msg_server_vote_inbound_tx.go

```diff
@@ -106,9 +106,12 @@ func (k msgServer) VoteInbound(
 	if err != nil {
 		return nil, err
 	}
-	// Process the inbound CCTX, the process function manages the state commit and cctx status change.
+	// Inititiate outbound, the process function manages the state commit and cctx status change.
 	//	If the process fails, the changes to the evm state are rolled back.
-	k.ProcessInbound(ctx, &cctx)
+	_, err = k.InitiateOutbound(ctx, &cctx)
+	if err != nil {
+		return nil, err
```

Comment:
Maybe we can wrap this err in a `ErrInitiatitingOutbound` to make it easier to pinpoint errors . What do you thing ?